pmg/templates/_create_alert.html

```diff
@@ -1,21 +1,31 @@
+{% set filtered_committee = [] %}
```

Comment:
How about we create a helper object in the search method in pmg/views.py that is passed to the template and has metadata such as the committee name, committee id, content type, friendly content type, year etc. That'll really make the views simpler.